javaparser-core/src/main/java/com/github/javaparser/Problem.java

```diff
@@ -69,21 +69,21 @@ public String getMessage() {
      * @return the message plus location information.
      */
     public String getVerboseMessage() {
-        return getLocation().map(l -> l.begin + " " + message).orElse(message);
+        return getLocation().map(l -> l.getBegin().getRange().begin + " " + message).orElse(message);
```

Comment:
Indeed, we now have...

tokenRange.begin.range.begin
tokenRange.begin.range.end
tokenRange.end.range.begin
tokenRange.end.range.end

For now the normal Range is still there.